openapi/go.xml

```diff
@@ -8,7 +8,7 @@
     <build>
         <plugins>
             <plugin>
-		<groupId>org.openapitools</groupId>
+		        <groupId>org.openapitools</groupId>
```

Comment:
This indentation is weird, please format. (here and below)

Reply:
The issue is that in that line there are 2 tabs while the rest of the file uses spaces. GitHub may be missleading, but the IDE comparision shows it better:

![image](https://user-images.githubusercontent.com/5781153/109721568-d1896a80-7bab-11eb-9c79-c477e77a4e99.png)

Reply:
Right, you need to be consistent with the rest of the file and use spaces.